sonic_platform_base/chassis_base.py

```diff
@@ -17,14 +17,14 @@ class ChassisBase(device_base.DeviceBase):
     DEVICE_TYPE = "chassis"
 
     # Possible reboot causes
-    REBOOT_CAUSE_POWER_LOSS = "power_loss"
-    REBOOT_CAUSE_THERMAL_OVERLOAD_CPU = "thermal_overload_cpu"
-    REBOOT_CAUSE_THERMAL_OVERLOAD_ASIC = "thermal_overload_asic"
-    REBOOT_CAUSE_THERMAL_OVERLOAD_OTHER = "thermal_overload_other"
-    REBOOT_CAUSE_INSUFFICIENT_FAN = "insufficient_fan"
-    REBOOT_CAUSE_WATCHDOG = "watchdog"
-    REBOOT_CAUSE_HARDWARE_OTHER = "hardware_other"
-    REBOOT_CAUSE_NON_HARDWARE = "non_hardware"
+    REBOOT_CAUSE_POWER_LOSS = "A/C Power Loss"
```

Comment:
>A/C  [](start = 31, length = 4)

Why specify 'A/C'?

Reply:
Just to make it more clear to anyone reviewing the logs that the A/C power was lost (i.e., an external power event).

Reply:
Changed to "Power Loss" to be more generic, in case power was lost due to a defective power supply, not loss of A/C.